modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/edit_role_permissions_summary.jsp

```diff
@@ -244,14 +244,21 @@
 			searchContainer="<%= searchContainer %>"
 			searchResultCssClass="show-quick-actions-on-hover table table-autofit"
 		/>
-
-		<c:if test="<%= searchContainer.getTotal() > 0 %>">
-			<div class="autofit-col sheet-footer taglib-search-iterator-page-iterator-bottom">
-				<liferay-ui:search-paginator
-					markupView="lexicon"
-					searchContainer="<%= searchContainer %>"
-				/>
-			</div>
-		</c:if>
-	</div>
-</div>
\ No newline at end of file
+	</clay:sheet-section>
+
+	<c:if test="<%= searchContainer.getTotal() > 0 %>">
+		<clay:sheet-footer>
+			<clay:content-row>
+				<clay:content-col
+					className="taglib-search-iterator-page-iterator-bottom"
```

Comment:
@jbalsas I'm still using `className` should I change it into `cssClass`?
https://github.com/brianchandotcom/liferay-portal/pull/89369/commits/532c1d8ba3610569df11056282b3392c4d79678e

/cc @marcoscv-work

Reply:
This pull will be merged after https://github.com/brianchandotcom/liferay-portal/commit/532c1d8ba3610569df11056282b3392c4d79678e
So I suppose new ones should be using `cssClass `

Reply:
Yes